src/controllers/MusicController.ts

```diff
@@ -297,6 +297,30 @@ export default class MusicController extends Controller {
 		}
 	};
 
+	/**
+	 *
+	 * @param songIndex is the index of the song to be moved.
+	 * @param newIndex is the new index of the song.
+	 */
+	move = (songIndex: number, newIndex: number) => {
+		let tempSong: Song;
+		const songs = this.queue;
+
+		// eslint-disable-next-line array-callback-return
```

Comment:
Do not ignore eslint rules, they're there for a reason. Return `false` at the end of the function's body instead.